pym/_emerge/depgraph.py

```diff
@@ -1539,8 +1539,7 @@ def __str__(self):
 		explored_nodes = set()
 
 		while unexplored:
-			# Handle all unexplored packages.
-			while unexplored:
+			while True: # Do {see below}
 				node = unexplored.pop()
```

Comment:
Maybe more readable like this:
```
try:
    node = unexplored.pop()
except IndexError:
    break
```

Reply:
https://pixel.nymag.com/imgs/daily/selectall/2017/02/13/wojak_01.nocrop.w710.h2147483647.jpg